Sources/PrimerSDK/Classes/Data Models/PrimerConfiguration.swift

```diff
@@ -37,6 +37,13 @@ struct PrimerAPIConfiguration: Codable {
             }
         }
         
+        #if !canImport(PrimerKlarnaSDK)
+        if let klarnaViewModelIndex = viewModels.firstIndex(where: { $0.config.type == PrimerPaymentMethodType.klarna.rawValue }) {
+            viewModels.remove(at: klarnaViewModelIndex)
+            print("WARNING!\nKlarna configuration has been found but module 'Primer/Klarna' is missing. Add `Primer/Klarna' in your project so you can perform payments with Klarna.")
```

Comment:
I guess the copy **might** be misleading. It seems like when we say `Primer/Klarna` we refer to a submodule of our PrimerSDK, while as far as I can see, it's an entirely different SDK, similar to Primer3DS.
I've also seen that we refer to it as a "module".
Considering the above my suggestion would be something like this:
```suggestion
            print("WARNING!\nKlarna configuration has been found, but the `PrimerKlarnaSDK` module is missing. Add `PrimerKlarnaSDK` via Pods or SwiftPackageManager in your project so you can perform payments with Klarna.")
```